internal/provisioner/plain/controllers/bundleinstance_controller.go

```diff
@@ -45,9 +45,11 @@ import (
 	"sigs.k8s.io/yaml"
 
 	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	crdanno "github.com/operator-framework/rukpak/cmd/crdvalidator/annotation"
 	helmpredicate "github.com/operator-framework/rukpak/internal/helm-operator-plugins/predicate"
 	"github.com/operator-framework/rukpak/internal/storage"
 	"github.com/operator-framework/rukpak/internal/util"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
nit: this import belongs with the 2nd block of package imports (e.g. stdlib, third party, internal)